pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -148,15 +148,6 @@ func NewEtcdCertSignerController(
 }
 
 func (c *EtcdCertSignerController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
```

Comment:
I think we definitely need to keep this check (in 4.16) because an update to the certificate bundle will also cause a revision rollout. 

You can test this with an earlier version as well, by adding a new entry to openshift-config/etcd-ca-bundle configmap.

Reply:
agree